packages/jest-worker/src/types.ts

```diff
@@ -19,6 +19,7 @@ export const CHILD_MESSAGE_END: 2 = 2;
 export const PARENT_MESSAGE_OK: 0 = 0;
 export const PARENT_MESSAGE_CLIENT_ERROR: 1 = 1;
 export const PARENT_MESSAGE_SETUP_ERROR: 2 = 2;
+export const PARENT_MESSAGE_CUSTOM: -1 = -1;
```

Comment:
any reason this is `-1` and not `3`?

Reply:
It was -1 when I cherry-picked the commit from Rogeliog, I thought there were some internal reasons to keep it the way it is.

If not, I'll change it back to 3 :D